src/systemctl/fuzz-systemctl-parse-argv.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+max_len = 65535
```

Comment:
It doesn't work with `AFL` and `honggfuzz`. Something like `if (size > 65535) return 0;` should be added to the fuzz target to cover those fuzzing engines as well.

Reply:
Oh, OK. I changed it to just refuse long command lines. ("Long" in the sense of number of entries. This is more useful than limiting size, because in this particular case we care about O(n²) in strv_append().)